### PR TITLE
chore: add newlines in status message...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: add newlines in status message MM_MESSAGE_APPEND so it's more readable

Change-Id: I8178472f618ea52f8e82d90a42b10a5f98124d2c
Signed-off-by: nickboldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by nickboldt** [comment]
As per change in https://github.com/eclipse/che/pull/20943#issuecomment-994815178 we can embed newlines in MM messages to make them more readable.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -135,7 +135,7 @@ jobs:
           export GH_PAGES_DIR="$PWD/gh-pages"
           ./build/dev/publish-plugin-registry-to-gh-pages.sh
 
-          echo MM_MESSAGE_APPEND='${{ env.MM_MESSAGE_APPEND }} Published package to npmjs https://www.npmjs.com/package/@eclipse-che/plugin-registry-generator/v/${{ github.event.inputs.version }}.' >> $GITHUB_ENV
+          echo MM_MESSAGE_APPEND='${{ env.MM_MESSAGE_APPEND }} \n Published package to npmjs https://www.npmjs.com/package/@eclipse-che/plugin-registry-generator/v/${{ github.event.inputs.version }}.' >> $GITHUB_ENV
```
Comment (mkuznyetsov) [inline .github/workflows/release.yml:138]:
```suggestion
          echo MM_MESSAGE_APPEND='${{ env.MM_MESSAGE_APPEND }} \n Published package to GitHub Pages https://github.com/eclipse-che/che-plugin-registry/tree/gh-pages/${{ github.event.inputs.version }}.' >> $GITHUB_ENV
```

**Review by mkuznyetsov — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (nickboldt) [reply .github/workflows/release.yml:138]: it's a new registry, not just a package.

Comment (nickboldt) [reply .github/workflows/release.yml:138]: `Published plugin registry to...`

**Review by svor — APPROVED** [review]

**Pull request merged**